### PR TITLE
Modify README.md and DeveloperGuide.md with simple use cases

### LINKED ISSUE #34: Update README.md
As per [Week 7's deliverables](https://nus-cs2103-ay2122s2.github.io/website/schedule/week7/project.html#2-update-project-website-aboutus-readme), to update README.md file and remove all traces of AB3

### PR DESCRIPTION
**Pull request by charmainehly**
Closes #34 and #27

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -287,28 +287,88 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+4.  TAlent Assistant™ adds the new TA to the list and displays the added entry
+
+    Use case ends.
+
+**Extensions**
+
+* 3a. The attribute fields added are invalid
+    * 3a1. TAlent Assistant™ shows an error message
+
+    Use case resumes at step 2.
```
Comment (domlimm) [inline docs/DeveloperGuide.md:322]:
```suggestion
1.  User requests to add a TA
2.  TAlent Assistant™ requests for TA's details
3.  User inputs details
4.  TAlent Assistant™ adds the new TA

    Use case ends.

**Extensions**

* 3a. The given format is invalid.

    * 3a1. AddressBook shows an error message.

      Use case resumes at step 2.
```

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -287,28 +287,88 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 (For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
 
-**Use case: Delete a person**
+**Use case: List all TA applicants**
```
Comment (domlimm) [inline docs/DeveloperGuide.md:290]: Charmaine this part, could do this instead:

You might need to add a new commit :') I let you decide!

```
1.  User requests to list TAs
2.  TAlent Assistant™ shows all TAs

    Use case ends.

**Extensions**

* 2a. The list is empty.

    * 2a1. TAlent Assistant™ feedbacks no TAs in system

      Use case ends.
```

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -287,28 +287,88 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+3.  User requests to delete an existing TA from the list
+4.  TAlent Assistant™ deletes the TA from the list and displays the deleted entry
+
+    Use case ends.
+
+**Extensions**
 
-* 3a. The given index is invalid.
+* 3a. The given index is invalid
+    * 3a1. TAlent Assistant™ shows an error message
```
Comment (domlimm) [inline docs/DeveloperGuide.md:338]:
```suggestion
* 3a. The given reference, i.e. Student Number, is invalid.

    * 3a1. TAlent Assistant™ shows an error message.
```

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -287,28 +287,88 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
 
-    * 3a1. AddressBook shows an error message.
+  Use case resumes at step 2.
+
```
Comment (domlimm) [inline docs/DeveloperGuide.md:341]: To beautify 💅🏻

```suggestion

<hr>

```

File: docs/DeveloperGuide.md [hunk 5]
```diff
@@ -287,28 +287,88 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+* 2a. No TA applicant entries contain the given keyword
+    * 2a1. TAlent Assistant™ returns an empty list with zero search results
+
+    Use case ends.
+
+* 2b. The specified attribute field is invalid
+    * 2b1. TAlent Assistant™ returns an error message
+
+    Use case resumes at step 1.
+
```
Comment (domlimm) [inline docs/DeveloperGuide.md:362]: To beautify 💅🏻

```suggestion

<hr>

```

File: docs/DeveloperGuide.md [hunk 6]
```diff
@@ -324,6 +384,7 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 * **Mainstream OS**: Windows, Linux, Unix, OS-X
 * **Private contact detail**: A contact detail that is not meant to be shared with others
+* **TA**: Teaching Assistant
```
Comment (domlimm) [inline docs/DeveloperGuide.md:387]: Nice! Think @LeeEnEn en en's doing this portion, seeing what he can add on.

**Review by domlimm — COMMENTED** [review]
Good work! Some suggestions to commit, let me know if there's anything! Thanks @charmainehly!

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (charmainehly) [reply docs/DeveloperGuide.md:293]: Have made and committed the change! Thanks for the suggestion!

**Pull request merged**